frontend/src/index.css

```diff
@@ -37,4 +37,9 @@ code {
   font-weight: 400;
   color: #FFFFFF !important;
   text-decoration: none
+}
+
+.log-entry h1{
+  font-size: 28px;
+  font-weight: 500;
 }
```

Comment:
some styling is missing, like the font colour, which should be #536C6E. The same applies to the icons. There is also an on-focus effect for the text inputs to change the border to #285E61.